src/common/store/synchronisation.ts

```diff
@@ -0,0 +1,42 @@
+import { Middleware } from '@reduxjs/toolkit';
+
+import { getHandshaken, getTargetPublicKey } from '@common/store/handshake';
+import { encryptJson } from '@common/utils';
+import { ReduxIPC } from '@types';
+
+/**
+ * Middleware that dispatches any actions to the other Electron process.
+ * @param ipc The Electron process to dispatch from.
+ */
+export const synchronisationMiddleware = (ipc: ReduxIPC): Middleware => (store) => (next) => (
+  action
+) => {
+  if (
```

Comment:
Should we have a better way of ignoring sensitive action types here? In the future we might want to ignore other things right?